proxy/proxy.go

```diff
@@ -404,6 +404,14 @@ func (proxy *Proxy) ContainerStarted(ident string) {
 	proxy.notifyWaiters(ident, err)
 }
 
+func (proxy *Proxy) ContainerConnected(ident string) {
+	err := proxy.attach(ident)
+	// if err != nil {
+	// TODO: Not sure what is needed here.
```

Comment:
Nothing.  We only expect container-connected events when going via the plugin, not the proxy.